articles/my-account-and-profile/sso.md

```diff
@@ -56,7 +56,7 @@ Metadata for US-hosted Customers:
 
 ```xml
 <?xml version="1.0"?>
-<EntityDescriptor xmlns="urn:oasis:names:tc:SAML:2.0:metadata" xmlns:ds="http://www.w3.org/2000/09/xmldsig# " entityID="passport-saml" ID="passport_saml">
+<EntityDescriptor xmlns="urn:oasis:names:tc:SAML:2.0:metadata" xmlns:ds="http://www.w3.org/2000/09/xmldsig# " entityID="https://api.sparkpost.com" ID="passport_saml">
```

Comment:
should we add a note somewhere that legacy customers may have an entityID of "passport-saml"?